sections/builder.js

```diff
@@ -1838,8 +1838,26 @@ export default [
   {
     type: "method",
     method: "connection",
-    description: " _ **(incomplete)** - This feature was incorrectly documented as functional._ <br/>If implemented, the method would set the db connection to use for the query without using the connection pool.",
-    children: [    ]
+    example: ".connection(dbConnection)",
+    description: "The method sets the db connection to use for the query without using the connection pool. You should pass to it the same object that acquireConnection() for the corresponding driver returns",
```

Comment:
Now that I think of this... We really should provide API for creating/destroying new connection object through knex which lives completely outside of the pool to make this feature complete and usable db agnostic way. Gotta remember to open feature request after I'm done with release stuff.

Reply:
Sure, I can implement that; do you have ideas how API could look?

Reply:
Great 👍 

I just left this comment here for me to remember to write feature request for that. I'll be writing API proposal there.

It should be easy to implement by just exposing methods that are currently used by pool to create / destroy connections.